sensor/Makefile

```diff
@@ -2,6 +2,9 @@
 bin/customsensor: *.go cmd/module/*.go
 	go build -o bin/customsensor cmd/module/cmd.go
 
+remote:
```

Comment:
This should probably be similar to the one above. E.g. a real file as the target, and deps on the go files involved, then just "remote" becomes a target that depends on the real-file target.

Reply:
I think I've fixed this now